### PR TITLE
Brush cache

### PR DESCRIPTION
**Pull request by BarryNolte**
Updated to include LRU functionality.

### CODE REVIEW

File: src/condo/SolidBrushCache.cs [hunk 1]
```diff
@@ -1,32 +1,75 @@
...
+
+            var newBr = new SolidColorBrush(new Color { R = R, G = G, B = B, A = 255 });
+            var brInfo = new BrushInfo { brush = newBr };
+
+            // If we are going to spill over our max count (count plus 1), then trim
+            if(this.cache.Count + 1 > maxCache)
+            {
+                Debug.WriteLine("Need to trim cache");
+
+                var x = this.cache
```
Comment (doubleyewdee) [inline src/condo/SolidBrushCache.cs]: So this approach doesn't feel right to me (it is not O(1) and can be). The general LRU impl would use the existing hashtable (dictionary) + a linked list to track LRU. I have most of that change ready and will submit a separate PR for you to look at.

**Comment by doubleyewdee** [comment]
I actually trialed this change out post-merge and it's introduced a pretty massive performance regression on my machine (oooops!) so I'm trying to track that down and figure out why that should be the case as the change is pretty simple.

File: src/condo/SolidBrushCache.cs (hunk 1 above)
Comment (BarryNolte) [reply src/condo/SolidBrushCache.cs]: I'll measure both and see who wins :).  The Debug.WriteLine probably bogs down debug builds under stress.

**Comment by doubleyewdee** [comment]
Merged a different change with same intent as #6
